helix-view/src/editor.rs

```diff
@@ -943,6 +930,8 @@ pub struct Editor {
     pub completion_request_handle: Option<oneshot::Sender<()>>,
 }
 
+pub type Motion = Arc<dyn Fn(&mut Editor)>;
```

Comment:
I don't think the Arc is necessary here, this can just be a Box as it was

Reply:
right 🙈